simulator-docs/src/main/asciidoc/manual/installation.adoc

```diff
@@ -136,6 +135,7 @@ See the following sample Maven project POM file that defines the project sources
   <version>1.0.0</version>
 
   <properties>
+    <java.version>11</java.version>
```

Comment:
The doc probably also should say Java 8, but this is debatable

Reply:
Done in 6bd41c1deb2a348d4f6dcee2f6e1f48c0376b6d3.